src/views/TimezoneModal.vue

```diff
@@ -158,4 +161,15 @@ export default defineComponent({
     };
   }
 });
-</script>
\ No newline at end of file
+</script>
+<style scoped>
+.spinner{
```

Comment:
No need to define custom css, try to find a way to center align the spinner with adding any custom css.